packages/react-core/src/components/AlertGroup/__tests__/__snapshots__/AlertGroup.test.tsx.snap

```diff
@@ -496,7 +498,7 @@ exports[`alertgroup closes when alerts are closed 1`] = `
                       aria-disabled={null}
                       aria-label="Close"
                       className="pf-c-button pf-m-plain"
-                      data-ouia-component-id={null}
+                      data-ouia-component-id={3}
```

Comment:
@redallen Why does this have `{}` around it and others not?

Reply:
Because it is not a string value. This is okay and expected!